package.json

```diff
@@ -31,6 +31,11 @@
       "type": "boolean",
       "title": "Enable Notice Messages",
       "default": false
+    },
+    "useProjectCwd": {
+      "type": "boolean",
+      "title": "Run Shellcheck using project root as the working directory (false for file-relative)",
```

Comment:
This should probably be the [`description`](https://atom.io/docs/api/v1.29.0/Config#title-and-description), although you may still want a `title` attribute to override the default of "Use Project Cwd" due to the "Cwd" part.

Reply:
Thanks for this feedback Landon, great point.  I've updated the metadata and here's what it looks like now:

<img width="748" alt="screen shot 2018-08-23 at 2 52 12 pm" src="https://user-images.githubusercontent.com/29980860/44545843-23e89700-a6e4-11e8-977f-0977b87e8d56.png">